Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py

```diff
@@ -610,12 +610,22 @@ def list_members_command(client, args):
     elif args.get('raw_json') is not None and kwargs:
         return_error("Please remove other arguments before using 'raw-json'.")
     response = client.list_members(**kwargs)
+    response['Members'] = convert_members_date_type(response.get('Members', []))
     outputs = {'AWS-SecurityHub': response}
     del response['ResponseMetadata']
-    table_header = 'AWS SecurityHub ListMembers'
+    table_header = 'AWS SecurityHub ListMembers' 
     human_readable = tableToMarkdown(table_header, response.get('Members', []))
     return human_readable, outputs, response
 
+def convert_members_date_type(members):
+    new_ls = []
+    for member in members:
+        if isinstance(updated_at:=member['UpdatedAt'], datetime):
+            member['UpdatedAt'] = updated_at.strftime('%Y-%m-%d %H:%M:%S')
+        if isinstance(invited_at:=member['InvitedAt'], datetime):
+            member['InvitedAt'] = invited_at.strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
```suggestion
        if isinstance(updated_at:=member.get('UpdatedAt'), datetime):
            member['UpdatedAt'] = updated_at.isoformat()
        if isinstance(invited_at:=member['InvitedAt'], datetime):
            member['InvitedAt'] = invited_at.isoformat()
```

Reply:
I think using **strftime** is better
**strdtime** yield this format: 2018-12-09 13:59:10
while **isoformat** yield this format: 2018-12-09T13:58:06.723000+00:00

Reply:
ISO format is the standard when it comes to time formats, it is the default format in the datetime library and it is also the default format that demisto is using.